BackendBench/eval.py

```diff
@@ -64,6 +66,21 @@ def allclose(a, b, atol=1e-2, rtol=1e-2):
         return False
 
 
+def equal_metadata(a, b):
```

Comment:
One thing I'm not super clear on is that OpInfo this is indeed the way they test tensor creation ops, that's how we figured out this might be the right testing strategy. So why not just use OpInfo again here?

Reply:
there is a reference here to pytorch's testing strategy https://github.com/pytorch/pytorch/blob/332fa5b388521c05a19217649745c6edfdc2836d/test/test_tensor_creation_ops.py#L3707